examples/speed_tester/speed_tester.c

```diff
@@ -334,6 +334,9 @@ run_advanced_rings(struct onvm_nf_info *nf_info) {
         tx_ring = nf->tx_q;
         msg_q = nf->msg_q;
 
+        /* Access the nf msg memory pool */
+        nf_msg_pool = rte_mempool_lookup(_NF_MSG_POOL_NAME);
+
```

Comment:
For future reference, prefer not to add to much white space. A good rule of thumb is to separate by newlines logical statements. Basic variable assignment shouldn't be separated